CMakeLists.txt

```diff
@@ -798,8 +798,8 @@ endif()
 ## Linux bundling of documentation + translation
 if(UNIX AND NOT APPLE)
   # install language help files
-  set(locales_help     en_EN de_DE es_ES fr_FR ja_JP pl_PL ru_RU uk_UA zh_CN zh_TW)
-  set(locales_help_msg en    de    es    fr    ja    pl    ru    uk    zh_CN zh_TW)
+  set(locales_help     en_EN de_DE es_ES fr_FR it_IT ja_JP pl_PL ru_RU uk_UA zh_CN zh_TW)
+  set(locales_help_msg en    de    es    fr    it    ja    pl    ru    uk    zh_CN zh_TW)
```

Comment:
there is the list ``all_locales`` that is returned from the call to ``generate_doc_targets``. Do you think it would be possible to link those two lists above to that ``all_locales`` list? There is the special case of ``zh`` that I did not know how to handle...